test/JDBC/expected/BABEL-2688-vu-prepare.out

```diff
@@ -0,0 +1,100 @@
+
+
+
+
+/* Example from JIRA */ 
+CREATE FUNCTION babel_2688_fn1() RETURNS @p TABLE (b bit)
+AS
+BEGIN
+            DECLARE @tv TABLE (b bit);
+            INSERT INTO @tv values (1);
+            UPDATE t
+            SET b  = 0
+            FROM @tv AS t;
+            INSERT INTO @p SELECT * FROM @tv;
+            RETURN;
+END
+GO
+
+CREATE TABLE babel_2688_tab1(a int)
+GO
+
+
+CREATE FUNCTION babel_2688_fn2() RETURNS int
+AS
+BEGIN
+	UPDATE t
+	SET a = 1
+	FROM babel_2688_tab1 AS t;
+	RETURN 0;
```

Comment:
I'm confused why this one has the error

Reply:
Normal tables shouldn't allow UPDATE, only table variables